### ISSUE TITLE
missing argument in polytools

### ISSUE THREAD

**Issue by frodo4fingers** [gimli-org/gimli#56]
hi

i am currently fooling around with a GUI for basic model creation. functions createRectangle() and createCircle() have the argument 'isHole' which i am missing in createPolygon(). i guess the implementation of the additional kwarg would a more consistent solution than writing that on my own...

**suggestion for createPolygon:**
```python
if isClosed and marker is not 0 or area > 0:
    if kwargs.pop('isHole', False):
        poly.addHoleMarker(pg.center(poly.positions()))
    else:
        poly.addRegionMarker(pg.center(poly.positions()), marker=marker, area=area)
```

since clicking around may result in a very abstract polygon figure the pg.center() function tends to calculate the center outside the polygon. for now i use [shapley](https://pypi.python.org/pypi/Shapely) but  i know that implementation of external libraries is difficult...

cheers :-)

**Comment by florian-wagner** [comment]
Hey Friedrich,

interesting stuff. A small mpl based gui for simple model creation would indeed be nice, especially for teaching. Since you are already familiar with the code, would you mind cloning pygimli, add the suggestion for createPolygon and open a (first!) pull request? The procedure is explained here: http://pygimli.org/contrib.html

Cheers
Florian

**Comment by florian-wagner** [comment]
closed by merging PR #57.
